src/main/java/com/contrastsecurity/rO0/RO0Agent.java

```diff
@@ -20,22 +34,81 @@ public static void agentmain(String args, Instrumentation inst) throws IOExcepti
 	private static void setup(String args, Instrumentation inst) {
 		ClassFileTransformer xform = new RO0Transformer();
 		inst.addTransformer(xform);
+		readEnvironment();
+	}
+	
+	private static void readEnvironment() {
+		
+		String outfile = System.getProperty("RO0.outfile");
```

Comment:
Yeah, that was a typo, and contradicts the README.  Will be fixed in my next submit.